Install racket 8.10 failed

### Provide environment information

```bash
OS:
Linux xxxtp 6.4.12-arch1-1 #1 SMP PREEMPT_DYNAMIC Thu, 24 Aug 2023 00:38:14 +0000 x86_64 GNU/Linux

SHELL:
zsh 5.9 (x86_64-pc-linux-gnu)

BASH VERSION:
5.1.16(1)-release

ASDF VERSION:
v0.13.1

ASDF INTERNAL VARIABLES:
ASDF_DEFAULT_TOOL_VERSIONS_FILENAME=.tool-versions
ASDF_DATA_DIR=/home/xxx/.asdf
ASDF_DIR=/opt/asdf-vm
ASDF_CONFIG_FILE=/home/xxx/.asdfrc

ASDF INSTALLED PLUGINS:
elixir                       https://github.com/asdf-vm/asdf-elixir.git master a4c42e1
erlang                       https://github.com/asdf-vm/asdf-erlang.git master 6744df7
golang                       https://github.com/asdf-community/asdf-golang.git master e7e8171
java                         https://github.com/halcyon/asdf-java.git master ebbae24
nodejs                       https://github.com/asdf-vm/asdf-nodejs.git master 9275f49
racket                       https://github.com/vic/asdf-racket.git master d90df36
rebar                        https://github.com/Stratus3D/asdf-rebar.git master 4af0770
ruby                         https://github.com/asdf-vm/asdf-ruby.git master 6fc8d03
scala                        https://github.com/mtatheonly/asdf-scala.git master 6de4edb
solidity                     https://github.com/diegodorado/asdf-solidity.git master cde20ca
```


### To Reproduce

run `asdf install racket 8.101

### Describe the Bug

Install log
```
configure: WARNING: you should use --build, --host, --target
configure: WARNING: invalid host type:  --prefix /home/xxx/.asdf/installs/racket/8.10
checking build system type... config.sub: invalid option --prefix
Try `config.sub --help' for more information.
configure: error: /bin/sh ../../lt/config.sub  --prefix /home/xxx/.asdf/installs/racket/8.10 failed
make: *** No targets specified and no makefile found.  Stop.
make: *** No rule to make target 'install'.  Stop.
/home/xxx/.asdf/plugins/racket/bin/install: line 46: /home/xxx/.asdf/installs/racket/8.10/bin/raco: No such file or directory
Installed Racket 8.10
```

After install, I can't run racket REPL.

### Expected Behaviour

instal racket correctly.

--- a/bin/install
+++ b/bin/install
@@ -38,7 +38,7 @@ racket_install() {
     curl -L "$(racket_pkg "$release")" >r"acket-${release}.tgz"
     tar -xf "racket-${release}.tgz"
     cd "racket-${release}/src/"
-    ./configure "${ASDF_RACKET_CONFIG_FLAGS}"
+    xargs ./configure <<<"${ASDF_RACKET_CONFIG_FLAGS}"
     make
     make install
 

